### DISCUSSION TITLE
”Use all my monitors for remote sessions“  setting does not work

### DISCUSSION THREAD

**Discussion by ljqlab** [rustdesk/rustdesk#8767] [General]
”Use all my monitors for remote sessions“  setting does not work, I still can't use two monitors at the same time when I'm remote.

**Comment by rustdesk** [comment]
the other sides must have >= your monitors

**Reply by ljqlab** [reply]
The remote is a server, it doesn't have a monitor. The client has two monitors, I want to use both monitors on the client to work together as the remote desktop monitor, is this not supported now?

**Reply by fufesou** [reply]
Yes, it is not supported for now.

It just uses existed displays.
It does not create virtual display for this option now.
